go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/pelletier/go-toml v1.9.4
 	github.com/rootless-containers/bypass4netns v0.2.2
 	github.com/rootless-containers/rootlesskit v1.0.0
+	github.com/schollz/progressbar/v3 v3.8.5
```

Comment:
Do we really need progressbar?

Reply:
Not an obligation but it is cool to have progress details?  WDYT ?

Reply:
Most copy operations won't take more than 3 seconds.
Not sure worth adding progressbar.

Reply:
fixed